docs/management_api.yml

```diff
@@ -68,52 +58,6 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
-  /users/inital:
```

Comment:
Looks like we'll need to increment API version increment this time. @maciejmrowiec comments/objections?